### PR TITLE
April 2, 2024 release notes

### PR DESCRIPTION
**Pull request by internetisaiah**
# Pull Request/Issue Resolution

#### Description of Change:

April 2, 2024 release notes, which includes:

- [x] [#7112](https://github.com/braze-inc/braze-docs/pull/7112) - Add WhatsApp product update for April 2024
- [x] [#7090](https://github.com/braze-inc/braze-docs/pull/7090) - Add integration instructions for Expo rich push and push stories for iOS
- [x] [#7108](https://github.com/braze-inc/braze-docs/pull/7108) - Add Shopify oAuth docs
- [x] [#7018](https://github.com/braze-inc/braze-docs/pull/7018) - Cdi s3 updates
- [x] [#6986](https://github.com/braze-inc/braze-docs/pull/6986) - Update cohort import page
- [x] [#7093](https://github.com/braze-inc/braze-docs/pull/7093) - Item recommendations GA
- [x] [#6930](https://github.com/braze-inc/braze-docs/pull/6930) - Live activities push to start
- [x] [#6955](https://github.com/braze-inc/braze-docs/pull/6955) - New WhatsApp article for multiple WABAs

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @internetisaiah as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Ana Teresa Serafino on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @internetisaiah for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@internetisaiah</td>
    <td>Developer Guide<br>Technology partners<br>Channels</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>Frontend Infrastructure & Experience (FIX)</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_help/release_notes/2024/4_2_24.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+nav_title: April 2, 2024
+description: "This article contains release notes for April 2, 2024."
+page_order: 9
+page_type: reference
+---
+
+# April 2, 2024 release
+
+## WhatsApp
```
Comment (internetisaiah) [inline _docs/_help/release_notes/2024/4_2_24.md:10]: I'm still not quite sure where this "WhatsApp" section should go? Or even if it should just be it's own section at the top like it is currently.

Comment (lydia-xie) [reply _docs/_help/release_notes/2024/4_2_24.md:10]: I think having this as its own section is okay

**Comment by internetisaiah** [comment]
**NOTE:** I still need to update the main landing page, but waiting till the content is reviewed.

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks great! can you also add the release notes at the top of here: https://www.braze.com/docs/help/release_notes and remove the release notes entry at the bottom?

**Comment by lydia-xie** [comment]
> **NOTE:** I still need to update the main landing page, but waiting till the content is reviewed.

Just saw this lol disregard my comment then!

File: _docs/_help/release_notes/2024/4_2_24.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Shopify  is a leading global commerce company providing trusted tools to start, grow, market, and manage a retail business of any size. Now when you set up Shopify for Braze, you can [enable OAuth for your workspace]({{site.baseurl}}/partners/message_orchestration/channel_extensions/ecommerce/shopify/setting_up_shopify/).
+
+### Use Expo for iOS push notifications
+
+We [added instructions]({{site.baseurl}}/developer_guide/platform_integration_guides/react_native/push_notifications/?tab=expo) for integrating rich push notifications and push stories into your iOS app using Expo with React Native.
+
+### Remote start iOS live activities
+
+Now you can remote start your live activities on iOS using the [`/messages/live_activity/start`]({{site.baseurl}}/api/endpoints/messaging/live_activity/start/) endpoint. For a full walkthrough, see [Live Activities: Start an Activity]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/live_activities/live_activities/#step-2-start-the-activity).
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2024/4_2_24.md:41]:
```suggestion
Now you can remote start your live activities on iOS using the [`/messages/live_activity/start` endpoint]({{site.baseurl}}/api/endpoints/messaging/live_activity/start/). For a full walkthrough, see [Live Activities: Start an Activity]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/live_activities/live_activities/#step-2-start-the-activity).
```

**Review by lydia-xie — APPROVED** [review]
marking as approved as to not block merging 👍

**Pull request merged**